src/AsyncStatusIndicator/AsyncStatusIndicator.d.ts

```diff
@@ -1,20 +1,6 @@
 import { CSSProperties, PureComponent } from 'react';
```

Comment:
this file should be completely removed: Typescript will automatically generate it